lib/VideoEncoder.js

```diff
@@ -64,8 +64,8 @@ class VideoEncoder extends Emitter
 		const scaleResolutionToHeight = params.scaleResolutionToHeight ? parseInt(params.scaleResolutionToHeight) : 0;
 		const allowedDownScaling = scaleResolutionToHeight && params.hasOwnProperty("allowSameHeight")? (params.allowSameHeight ? 1 : 2) : 0;
 		//Get default for width and heights
-		const width  = parseInt(params.width)  ? parseInt(params.width)  : (scaleResolutionDownBy ? 640 : 0);
-		const height = parseInt(params.height) ? parseInt(params.height) : (scaleResolutionDownBy ? 480 : 0);
+		const width  = params.width  ? parseInt(params.width)  : (scaleResolutionDownBy ? 640 : 0);
```

Comment:
we need to sanitize  input data, iirc swig crashes if wrong type is used
```suggestion
		const width  = parseInt(params.width)  ?? 0;
```

Reply:
your suggestion removes the default of 640, are you sure we want that?

Reply:
I think that should be the ultimate goal, we can be consertative and just do this instead

```suggestion
		const width  = parseInt(params.width)  ?? (scaleResolutionDownBy ? 640 : 0);
```